resources/js/package.js

```diff
@@ -103,6 +103,10 @@ function init() {
             setSearchParams(url) {
                 window.history.pushState(window.history.state, '', new URL(url))
             },
+            toggleScroll(bool) {
```

Comment:
It might be a nice idea to implement https://vueuse.org/core/useScrollLock/ so locked state can be read as well as written, and it seems to have a fix for IOS supposedly not listening to scroll locks properly